docs/pages/eas-update/introduction.mdx

```diff
@@ -17,19 +17,19 @@ Still using Classic Updates? We moved those docs to [the archive](/archive/class
 
 ## Pitch
 
-### Improve User Experience
+### Improve user experience
 
 EAS Update allows you to release hotfixes and enhancements in response to user feedback or market trends. It helps you keep users engaged and satisfied with your app. Don't wait for the next planned update or manual update request.
 
-### Broad Compatibility
+### Broad compatibility
 
 EAS Update works with vanilla React Native apps. All you need to do is install the lightweight `expo` npm package as a prerequisite, making it a compelling alternative to solutions such as [Codepush](/eas-update/codepush).
 
 ### Tailored for Expo
 
 EAS Update is designed for Expo apps, making integration easy and workflows efficient. Use the website UIs to check app status and debug, especially for developers using EAS Build.
 
-### Customizable Update Strategies
+### Customizable Update strategies
```

Comment:
```suggestion
### Customizable update strategies
```

Shouldn't "Update" also be lowercased in this sentence?

Reply:
Yes good catch!